README.md

```diff
@@ -3,6 +3,7 @@
 A collection of examples to enable new users to start using ROCm. Advanced users may learn about new functionality through our advanced examples.
 
 ## Repository Contents
+- [AI]
```

Comment:
you're missing the link here

Reply:
## Repository Contents
- [AI](https://github.com/rocm-examples/tree/develop/AI/MIGraphX/Quantization) Showcases the functionality for executing quantized models using MIGraphX.

https://github.com/Rmalavally/rocm-examples/blob/Rmalavally-patch-2-INT8-ROCm6.1/README.md